src/transformers/models/clip/feature_extraction_clip.py

```diff
@@ -142,11 +146,13 @@ def __call__(
         if not is_batched:
             images = [images]
 
-        # transformations (resizing + center cropping + normalization)
+        # transformations (resizing + center cropping + convert rgb + normalization)
         if self.do_resize and self.size is not None and self.resample is not None:
             images = [self.resize(image=image, size=self.size, resample=self.resample) for image in images]
         if self.do_center_crop and self.crop_size is not None:
             images = [self.center_crop(image, self.crop_size) for image in images]
+        if self.do_convert_rgb:
+            images = [self.convert_rgb(image) for image in images]
```

Comment:
This should be at the very beginning of the pipeline I think, even before the resize.